testing/simapp/app.go

```diff
@@ -313,31 +326,38 @@ func NewSimApp(
 	// }
 	// baseAppOptions = append(baseAppOptions, prepareOpt)
 
+	// create and set dummy vote extension handler
```

Comment:
@tac0turtle - do you think that we need this?  It is why I had abci.go in the repository.

Reply:
for now, I am going to comment it out.